### PR TITLE
Add routing

### PR DESCRIPTION
**Pull request by j-tee**
Built todos app in react

### CODE REVIEW

File: src/classBased/components/InputTodo.js [hunk 1]
```diff
@@ -0,0 +1,47 @@
+/* eslint-disable no-alert */
+/* eslint-disable react/button-has-type */
+/* eslint-disable react/prop-types */
+/* eslint-disable react/destructuring-assignment */
+/* eslint-disable react/state-in-constructor */
```
Comment (ouasamine) [inline src/classBased/components/InputTodo.js:5]: - Please try to limit the use of linters disabling comments, respecting linters' directives improve your code quality in general.

**Review by ouasamine — APPROVED** [review]
Hi @j-tee, your project is fully functional, congratulations on your hard work you have done very well 👏🏽 👏🏽 

## Highlights

- No linter errors ✔️ 
- Descriptive commits ✔️ 
- Correct Gitflow used ✔️ 

## Suggestions

These are some small changes that would make your project perfect:
- Using a professional README based on Microverse [template](https://github.com/microverseinc/readme-template).
- Changing your React app title to a more descriptive one by editing the `<title>` tag inside `public/index.html`.

Please find some additional suggestions below.

Your code is modular and professional, keep up the good work 💯 👍🏽 👍🏽

File: src/classBased/components/InputTodo.js (hunk 1 above)
Comment (j-tee) [reply src/classBased/components/InputTodo.js:5]: OK, Changes completed

**Comment by j-tee** [comment]
Thanks for the feedback!!

**Pull request merged**